exercises/clock/clock_test.rb

```diff
@@ -5,281 +5,313 @@
 class ClockTest < Minitest::Test
   def test_on_the_hour
     # skip
-    assert_equal "08:00", Clock.at(8, 0).to_s
+    assert_equal "08:00", Clock.new(hour: 8, minute: 0).to_s
   end
 
   def test_past_the_hour
     skip
-    assert_equal "11:09", Clock.at(11, 9).to_s
+    assert_equal "11:09", Clock.new(hour: 11, minute: 9).to_s
   end
 
   def test_midnight_is_zero_hours
     skip
-    assert_equal "00:00", Clock.at(24, 0).to_s
+    assert_equal "00:00", Clock.new(hour: 24, minute: 0).to_s
   end
 
   def test_hour_rolls_over
     skip
-    assert_equal "01:00", Clock.at(25, 0).to_s
+    assert_equal "01:00", Clock.new(hour: 25, minute: 0).to_s
   end
 
   def test_hour_rolls_over_continuously
     skip
-    assert_equal "04:00", Clock.at(100, 0).to_s
+    assert_equal "04:00", Clock.new(hour: 100, minute: 0).to_s
   end
 
   def test_sixty_minutes_is_next_hour
     skip
-    assert_equal "02:00", Clock.at(1, 60).to_s
+    assert_equal "02:00", Clock.new(hour: 1, minute: 60).to_s
   end
 
   def test_minutes_roll_over
     skip
-    assert_equal "02:40", Clock.at(0, 160).to_s
+    assert_equal "02:40", Clock.new(hour: 0, minute: 160).to_s
```

Comment:
I think in this first section we should leave out the `hour:` or `minute:` argument when its value is zero.
Ideally we'd have separate tests for default keyword values, but sneaking the requirement in this way gets it done without having to do anything too special to the canonical tests. And it doesn't suddenly come as a huge surprise when we hit the `add` tests.

Reply:
I like that idea.

Reply:
Do you only want it left out for the tests where `property == 'create'` ?
Or should they be left out for all of them?